### PR TITLE
Milestone: 6 Website Accessibility Check

### PR DESCRIPTION
**Pull request by muhammedb8826**
## Accessibility Checking

- Title- changed to my [page-title](https://github.com/muhammedb8826/Portfolio-setup-and-mobile-first/pull/8/files)
- Image text alternatives- Improved on the first [alt-text](https://github.com/muhammedb8826/Portfolio-setup-and-mobile-first/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R44-R50)
- Text heading- No issues with text headings
- Color contrast- [color-contrast](https://github.com/muhammedb8826/Portfolio-setup-and-mobile-first/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR31-R49) 
- Resize- No issues with resize
- Label- Added hidden form labels on the second [label-text](https://github.com/muhammedb8826/Portfolio-setup-and-mobile-first/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R262-R282)
- Moving content- No issues
- Multimedia- No issues
- Structure- No issues with the structure

### CODE REVIEW

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @muhammedb8826 , @ClintonYounge , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
-  Kindly, the title has to adequately and briefly describes the content of the page. For example: `Mohammed portfolio`👌for better accessibility 👍
-  Kindly, It would be better to implement `hover` and `focus` state for all your links to be easily identified 👍👌 Your logo hasn't have any hover and focuses so it fails color contrast, please fix it
 Screenshot_20230125_120946
- [Optional] Kindly, I think it would be better relative measurements like `em`, `rem` instead of absolute measurements, to make your content resizable 👌👍

## Optional suggestions
_Every comment with the [
OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @muhammedb8826 @ClintonYounge 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please follow the previous reviewer's advice on fixing the color contrast of your title. At the moment it isn't passing the contrast check, you can use [this tool](https://www.aditus.io/button-contrast-checker/) to check the color contrast before submitting your project again. Remember that having a good contrast will make your site easier to read for those who have reduced visibility! 🤓 🔧 

image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ClintonYounge** [comment]
@muhammedb8826 Message me on Slack.

**Review by IngDavidTM — APPROVED** [review]
Hi @muhammedb8826 and @ClintonYounge   🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio-setup-and-mobile-first</title>
+    <title>Muhammed's Portfolio</title>
```
Comment (muhammedb8826) [inline index.html:7]: Title changed to my own name

File: index.html [hunk 2]
```diff
@@ -41,13 +41,13 @@ <h3 class="social-media">LET’S CONNECT</h3>
...
           <li>
-            <a href="#"><img src="./assets/Vector (2).svg" alt="m" /></a>
+            <a href="#"><img src="./assets/Vector (2).svg" alt="medium" /></a>
           </li>
           <li>
             <a href="#"><img src="./assets/Vector (1).svg" alt="github" /></a>
           </li>
           <li>
-            <a href="#"><img src="./assets/Vector (3).svg" alt="s" /></a>
+            <a href="#"><img src="./assets/Vector (3).svg" alt="angelist" /></a>
```
Comment (muhammedb8826) [inline index.html:50]: Alt text improved for medium and angelist icons

File: index.html [hunk 3]
```diff
@@ -257,19 +259,24 @@ <h2>Contact me</h2>
...
             type="email"
             placeholder="natajayanj14@gmail.com"
+            id="email"
             class="input-text"
             required
           />
+          <label for="form-text-area" hidden>what's your message</label>
           <textarea
             name="form-description"
             id="form-text-area"
```
Comment (muhammedb8826) [inline index.html:282]: Label text added to improve accessibilty

**Comment by muhammedb8826** [comment]
Link added to pull request description

**Review by muhammedb8826 — COMMENTED** [review]
Accessibility issues improved to make the website more accessible

**Review by muhammedb8826 — COMMENTED** [review]
All color contrast is fixed to improve its accessibility

photo_2023-01-25_03-30-43

File: style.css [hunk 4]
```diff
@@ -28,24 +28,25 @@ body {
...
+  color: #4d5bd9;
+}
+
+.header > a:active {
+  color: #2230d2;
+  border-radius: 8px;
+}
+
+.header > a:disabled {
+  color: #5e6c84;
```
Comment (muhammedb8826) [inline style.css:49]: Color contrast issues fixed, to improve accessibility of the web page

**Review by muhammedb8826 — COMMENTED** [review]
Color contrast fixed

**Comment by muhammedb8826** [comment]
Color contrast fixed

**Pull request merged**